Base/res/html/tests/Attr-cloneNode.html

```diff
@@ -0,0 +1,34 @@
+<html>
+<body>
+  <pre id="out"></pre>
+  <script>
+    function log(s) {
+      document.getElementById("out").innerHTML += s + "\n"
+    }
+
+    window.onload = function() {
+      let e = document.createElement("div")
+      e.setAttribute("foo", "bar")
+      let attr = e.getAttributeNode("foo")
+      let clone = attr.cloneNode()
+
+      function dumpAttr(name, attr) {
+        log(name + ": " + attr)
+        log(name + ".ownerElement: " + attr.ownerElement)
+        log(name + ".namespaceURI: " + attr.namespaceURI)
+        log(name + ".localName: " + attr.localName)
+        log(name + ".name: " + attr.name)
+        log(name + ".value: " + attr.value)
+        log(name + ".specified: " + attr.specified)
+        log("")
+      }
+
+      dumpAttr("attr", attr)
+      dumpAttr("clone", clone)
+
+      log("attr === clone -> " + attr === clone)
```

Comment:
Small detail, these will always return false because `"attr === clone -> " + attr` will be evaluated first and turn the LHS into a string, then `=== clone` will be evaluated second and always be false! This can be seen in the video, as the the log lines were written as `false` instead of `attr === clone -> false`

```suggestion
      log("attr === clone -> " + (attr === clone))
```